docs/releases/4.1.txt

```diff
@@ -323,6 +323,17 @@ Miscellaneous
 
 * ``CSRF_COOKIE_MASKED`` transitional setting is deprecated.
 
+* The API of :meth:`~django.test.SimpleTestCase.assertFormError()` and
+  :meth:`~django.test.SimpleTestCase.assertFormsetError()` have changed.
```

Comment:
Write in present tense ("is" instead of "haven been", "was", etc.)

Reply:
Thanks 👍🏻